+neuron/+stack/hoc_pop.m

```diff
@@ -7,7 +7,12 @@
         value = clib.neuron.matlab_hoc_strpop();
     elseif (returntype=="Object")
         obj = clib.neuron.matlab_hoc_objpop();
-        value = neuron.Object(obj);
+        objtype = obj.ctemplate.sym.name;
+        if objtype == "Vector"
+            value = neuron.Vector(obj);
```

Comment:
we have two other specialized subtypes of `neuron.Object`; do we need to do the same for `PlotShape` and `RangeVarPlot`? If not, why not?

Reply:
You are right, we need to do the same for those.